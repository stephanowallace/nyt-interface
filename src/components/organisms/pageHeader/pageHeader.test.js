import React from 'react';
import { shallow } from 'enzyme';
import PageHeader from '.';

describe('PageHeader', () => {
  it('renders default state', () => {
    const component = shallow(<PageHeader title="test" />);
    expect(component).toMatchSnapshot();
  });
});
