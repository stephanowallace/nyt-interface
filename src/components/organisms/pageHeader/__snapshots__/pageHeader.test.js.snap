// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders default state 1`] = `
<Fragment>
  <SectionTitle
    className=""
  >
    test
  </SectionTitle>
  <withRouter(Navigation) />
</Fragment>
`;
